Sources/ArcGISToolkit/Common/AttachmentPreview.swift

```diff
@@ -77,8 +85,20 @@ struct AttachmentPreview: View {
             }
         }
         .alert("Rename attachment", isPresented: $renameDialogueIsShowing) {
-            if let onRename, let renamedAttachmentModel {
-                AttachmentRenameAlert(attachmentModel: renamedAttachmentModel, onRename: onRename)
+            TextField("New name", text: $newAttachmentName)
+            Button("Cancel", role: .cancel) { }
+            Button("Ok") {
```

Comment:
Do we need to localize these? And I think "Ok" should be all caps: "OK". That string exists in the localizable.strings files.

Reply:
It would be nice if there was a "clear" (`x`) button in the text filed, but that can come later, unless it's easy to do now.

Reply:
- Changed "Ok" to "OK".
- Yes we need to localize but I'd prefer to do that as a separate PR because I believe there're other non rename related strings we've since added that need to be done too and generating the strings file is a bit involved. I can add a bullet for this in the Apollo issue and do this next.

Reply:
- Alerts seems to be very restrictive on how content is structured. A button placed into a `ZStack` with the `TextField` just ends up appearing inline with the cancel and OK buttons. Even in an `.overlay` on the `TextField` the same thing happens. If we want to pursue this we might need to do away with the alert.

Reply:
Ok, no worries, we can leave it as-is.